doc/cookbook/source/examples/classifier/mkl_binary_class.rst

```diff
@@ -0,0 +1,55 @@
+==================================================
+Multiple Kernel Learning for Binary Classification
+==================================================
+
+Multiple kernel learning (MKL) is based on convex combinations of arbitrary kernels over different domains. One could make a combination of kernels like:
+
+.. math::
+
+    {\bf k}(x_i,x_j)=\sum_{i=1}^{K} \beta_k {\bf k}_i(x_i, x_j)
+
+where :math:`\beta_k > 0`, :math:`\sum_{k=1}^{K} \beta_k = 1`, :math:`K` is the number of sub-kernels, :math:`\bf{k}` is a combined kernel matrix, :math:`{\bf k}_i` is an individual kernel matrix and :math:`x_i` is a data point.
+
+Classification is done by using Support Vector Machines (SVM). `See the linear SVM cookbook <http://shogun.ml/cookbook/latest/examples/classifier/linear_svm.html>`_ for more details. Optimal :math:`\alpha` and :math:`b` for SVM and :math:`\beta` are determined via training.
+
+See :cite:`sonnenburg2006large` for more details.
+
+-------
+Example
+-------
+
+Imagine we have files with training and test data. We create CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CBinaryLabels` as
+
+.. sgexample:: mkl_binary_class.sg:create_features
+
+Then we create indvidual kernels like :sgclass:`CPolyKernel`, :sgclass:`CCustomKernel` and :sgclass:`CGaussianKernel` which will be later combined in one :sgclass:`CCombinedKernel`.
+
+.. sgexample:: mkl_binary_class.sg:create_kernel
+
+We create an instance of :sgclass:`CCombinedKernel` and append the :sgclass:`CKernel` objects. In order to initialize :sgclass:`CCombineKernel` with features, we also need to create an object of :sgclass:`CCombinedFeatures`.
```

Comment:
The combined features are for the case where different kernels have different features.
You dont need to do that here, but mention it. Also you combined features only get two copies of the training features here. Is that because one kernel is precomputed?